### PR TITLE
about page dropdown

### PR DESCRIPTION
**Pull request by JustinStitt**
## Old Behavior
dropdown-old
**notice the jumps and ability to click __invisible__ options!**

## New Behavior
dropdown
**much nicer 😄**

### Also added new on hover effect for team members
hover1

### CODE REVIEW

File: src/lib/components/select/select.svelte [hunk 1]
```diff
@@ -22,28 +19,40 @@
...
-        <div
-          class="option-choice"
-          on:click={(ev) => handleOption(ev, optionValue)}
-          on:keypress={(ev) => handleOption(ev, optionValue)}
-        >
-          {optionValue}
-        </div>
-      {/each}
-    </div>
+    {#if active === true}
```
Comment (jaasonw) [inline src/lib/components/select/select.svelte]: not pep8 compliant

**Review by jaasonw — CHANGES_REQUESTED** [review]

File: src/lib/components/select/select.svelte (hunk 1 above)
Comment (jjoeldaniel) [reply src/lib/components/select/select.svelte]: fixed [here](https://github.com/EthanThatOneKid/acmcsuf.com/pull/842/commits/d05b4998f300e967101fceacfd7cbd221dd2113a)

File: src/routes/(site)/wave1/+page.svelte [hunk 2]
```diff
@@ -8,9 +8,12 @@
   import PositionList from './position-list.svelte';
   import { POSITIONS } from './data';
 
+  let expanded = false;
   function expandAll() {
     const positions = document.querySelectorAll('.position');
-    positions.forEach((el) => el.setAttribute('open', 'true'));
+    if (expanded === true) positions.forEach((el) => el.removeAttribute('open'));
```
Comment (jaasonw) [inline src/routes/(site)/wave1/+page.svelte]: nortt pep8 compilkert

**Review by jaasonw — DISMISSED** [review]

File: src/routes/(site)/wave1/+page.svelte (hunk 2 above)
Comment (jjoeldaniel) [reply src/routes/(site)/wave1/+page.svelte]: resolved [here](https://github.com/EthanThatOneKid/acmcsuf.com/pull/842/commits/2ae8197a84e81263754fb19d6fbcc088778b0255)

**Review by jjoeldaniel — APPROVED** [review]
lgtm - the j's

**Comment by jaasonw** [comment]
ur a stale revirew

**Comment by k4rni** [comment]
Thank you for catching this and for the PR.

Make sure to check what files you're committing next time because you pushed some changes into prod that weren't approved yet!

**Pull request merged**